README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Routing Number</span>
+            <input type="text" size="10" bf-data="routingNumber" value="110000000"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Account Number</span>
+            <input type="text" size="10" bf-data="accountNumber" value="000123456789"/>
+        </label>
+    </div>
+
+    <div class="form-row">
+        <label>
+            <span>Account Holder Type</span>
+            <select bf-data="accountHolderType">
+                <option>individual</option>
+                <option>company</option>
+            </select>
+        </label>
+    </div>
+
+    <input type="hidden" bf-data="accountID" value="SET AS ACCOUNT ID" />
+
+    <button type="submit">Submit Payment</button>
+</form>
+```
+
+The form is comprised of the following fields:
+
+- **holderName:** the account holder name.
+- **bankAccountName:** a label identifying the bank account
+- **routingNumber:** the routing number
+- **accountNumber:**  the account number
+- **accountHolderType:** the holder type (accepted values are 'individual' and 'company')
+- **accountID:** the accound's id we are adding the bank account to
+
+A small amount of `javascript` is needed as well to bootstrap correctly the `html form`:
+
+```javascript
+        var bfAPIKey = 'YOUR BILLFORWARD PUBLIC TOKEN HERE';
+        var bfAPIURL = 'https://api-sandbox.billforward.net:443/v1/';
+
+        // connect to BillForward using credentials
+        BillForward.useAPI(bfAPIURL, bfAPIKey);
+
+        // use a 'null' account if you want to capture this card against a new customer
+        var accountID = null;
+
+        // Jquery-style selector pointing to your form
+        var formSelector = '#payment-form';
+
+        // Jquery-style selector pointing to your verify form
+        var verifyFormSelector = '#verify-form';
+
+        BillForward.captureBankAccountOnSubmit(formSelector, 'stripe', accountID, callback);
+
+        function callback(paymentMethod, error) {
+            if (error) {
+                console.error(error);
+            } else {
+                console.log(paymentMethod);
+
+                // Do some stuff once the capture is complete
+            }
+        }
+```
+
+#### Capture directly with code
+
+It is possible to capture a bank account directly with javascript without the usage of an `html form`.
+For example:
+
+```javascript
+var bankDetails = {
+    "routingNumber":"110000000",
+    "accountNumber":"000123456789",
+    "holderName":"John Doe",
+    "bankAccountName":"John Doe's Bank",
+    "accountHolderType":"company"
+};
+
+BillForward.captureBankAccount(bankDetails, 'stripe', accountID, callback);
+
+```
+
+#### Verify with a form
+
+After we have successfully created a bank account we have to verify it to be able to use it for a payment. The way `stripe` verify its bank accounts is through `micro deposits`.
```

Comment:
`stripe` -> Stripe

These need to be separate paragraphs, or at least separate lines. They are conceptually different steps, so the reader needs to be able to skip to the next one by looking at the next line.

In GitHub markdown (at least), you can:
- start a new paragraph using two line breaks.
- start a new line by ending your current line with two spaces (`  `).